src/components/nav/NavBar.js

```diff
@@ -30,35 +58,42 @@ export const NavBar = ({ token, setToken }) => {
 
       <div className="navbar-menu" ref={navbar}>
         <div className="navbar-start">
-          {
-            token
-              ?
-              <Link to="/" className="navbar-item">Posts</Link>
-              :
-              ""
-          }
+          {token ? (
```

Comment:
Please add a comment explaining what this ternary statement is doing

Reply:
done 👍